io.catenax.mandatory_dismantling/1.0.0/MandatoryDismantling.ttl

```diff
@@ -0,0 +1,994 @@
+   samm:properties ( :electricVehicleBattery :eDriveMotor :sliBattery :engine :catalyticConverter :gearbox :window :wheel :tyre :dashboard :infotainmentSystem :headlight :wireHarness :bumper :fluidContainer :heatExchanger :metalComponent :plasticComponent :inverter :printedCircuitBoard :pvPanel :transmissionComponent ) .
+
+:pollutant a samm:Property ;
+   samm:preferredName "Pollutant"@en ;
+   samm:description "Type of pollutant present. \"Pb\" (Lead), \"Hg\" (Mercury), \"Cd\" (Cadmium), \"Cr\" (Chromium), etc."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Pb" .
+
+:number a samm:Property ;
+   samm:preferredName "Number"@en ;
+   samm:description "Number of specific pollutant, component or part."@en ;
+   samm:characteristic :QuantityCharacteristic ;
+   samm:exampleValue 1 .
+
+:part a samm:Property ;
+   samm:preferredName "Part"@en ;
+   samm:description "The part or component that contains the specific pollutant."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Starter battery" .
+
+:location a samm:Property ;
+   samm:preferredName "Location"@en ;
+   samm:description "String specifying the location of the pollutant , component or part."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Front Right Panel" .
+
+:weight a samm:Property ;
+   samm:preferredName "Weight"@en ;
+   samm:description "Weight of the part that contains the specific pollutant in grams"@en ;
+   samm:characteristic :WeightCharacteristic ;
+   samm:exampleValue 20 .
+
+:minimumConcentration a samm:Property ;
+   samm:preferredName "Minimum Concentration"@en ;
+   samm:description "Minimum weight percentage of pollutant concentration to be dismantled"@en ;
+   samm:characteristic :ConcentrationCharacteristic ;
+   samm:exampleValue "0.4"^^xsd:float .
+
+:pollutantConcentration a samm:Property ;
+   samm:preferredName "Pollutant Concentration"@en ;
+   samm:description "weight percentage of pollutant concentration"@en ;
+   samm:characteristic :ConcentrationCharacteristic ;
+   samm:exampleValue "0.5"^^xsd:float .
+
+:weightOfPart a samm:Property ;
+   samm:preferredName "Weight Of Part"@en ;
+   samm:description "Weight of part containing pollutant in gram"@en ;
+   samm:characteristic :WeightCharacteristic ;
+   samm:exampleValue 500 .
+
+:fluid a samm:Property ;
+   samm:preferredName "Fluid"@en ;
+   samm:description "Information on the different fluid and liquid available in the asset"@en ;
+   samm:characteristic :FluidCharacteristic .
+
+:component a samm:Property ;
+   samm:preferredName "Component"@en ;
+   samm:description "information on the different components available in the asset"@en ;
+   samm:characteristic :ComponentCharacteristic .
+
+:time a samm:Property ;
+   samm:preferredName "Time"@en ;
+   samm:description "Time of the specific depollution."@en ;
+   samm:characteristic :Time ;
+   samm:exampleValue "14:34:00"^^xsd:time .
+
+:date a samm:Property ;
```

Comment:
Property and the referenced Characteristic should not have the same name

Reply:
Fixed